docs/apache-airflow/macros-ref.rst

```diff
@@ -62,7 +62,7 @@ Variable                                Description
 ``{{ ti }}``                            same as ``{{ task_instance }}``
 ``{{ params }}``                        a reference to the user-defined params dictionary which can be overridden by
                                         the dictionary passed through ``trigger_dag -c`` if you enabled
-                                        ``dag_run_conf_overrides_params` in ``airflow.cfg``
```

Comment:
Currently how this is rendered, which is obviously wrong

<img width="779" alt="Macros_reference_—_Airflow_Documentation" src="https://user-images.githubusercontent.com/11539188/111885647-67553000-89c9-11eb-822f-77d7de4fb212.png">